WcaOnRails/app/assets/javascripts/registrations.js

```diff
@@ -21,7 +21,8 @@ onPage('registrations#edit_registrations', function() {
   showHideActions();
 
   $('button[value=delete-selected]').on("click", function(e) {
-    if(!confirm("Are you sure you want to delete the selected registrations?")) {
+    var $selectedRows = $registrationsTable.find("tr.selected");
+    if(!confirm("You are about to delete " + $selectedRows.length + " registrations.\nAre you sure you want to do it?")) {
```

Comment:
I think shorter is better:

`"Are you sure you want to delete the " + $selectedRows.length + "  selected registrations?"`